hack/ssh/key

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAABlwAAAAdzc2gtcn
```

Comment:
This may have been useful for testing, but I don't think we want to commit a dummy SSH key to the installer repo.

Reply:
same